### PR TITLE
Polymer 3 Setup

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: demo/siren-entity-demo.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<link rel="import" href="../../polymer/polymer.html">
```
Comment (jwong-d2l) [inline demo/siren-entity-demo.html]: Is the polymer3 pattern for demos to be their own components now?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "test": "polymer test --skip-plugin sauce",
-    "postinstall": "polymer install"
+    "postinstall": "polymer install",
+    "demo": "polymer serve"
```
Comment (dlockhart) [inline package.json]: License should be `Apache-2.0`.

File: .travis.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 9
+node_js: node
```
Comment (dlockhart) [inline .travis.yml]: Since you're not doing any automated testing in Sauce Labs, I don't think you need the `chrome` addon?

Comment (dlockhart) [reply .travis.yml]: `sudo: required` probably also isn't... required.

File: demo/siren-entity-demo.html (hunk 1 above)
Comment (ChrisKraljevicD2L) [reply demo/siren-entity-demo.html]: Not that I know of, but this change was to fix the demo. Data binding in the original demo was not working (see #5 ) so I thew this together to interface with `siren-entity`.

**Pull request merged**